static/sass/_charmhub_releases.scss

```diff
@@ -29,28 +29,19 @@
       box-shadow: 0 3px 3px rgba(51, 51, 51, 0.2);
       display: flex;
       flex-direction: column;
-      margin: $sp-small 0 $sp-small 0;
       max-height: 250px;
       min-width: 150px;
       order: 1;
       overflow-y: auto;
       position: absolute;
       width: auto;
-      z-index: 1000;
+      z-index: 5;
 
     @media only screen and (max-width: $breakpoint-large) {
       width: 96%;
     }
   }
-
-  .dropdown-menu.padding-bottom {
-    padding: $sp-small;
-  }
   
-  .dropdown-menu.no-padding-bottom {
-    padding: $sp-small $sp-small 0 $sp-small;
-  }
-
   .dropdown-item {
```

Comment:
Can you add padding to the dropdown items on hover like I mentioned in your last PR?